ext/xmlwriter/tests/002.phpt

```diff
@@ -5,7 +5,7 @@ XMLWriter: libxml2 XML Writer, membuffer, flush
 --FILE--
 <?php
 
-$doc_dest = '001.xml';
+$doc_dest = '002.xml';
```

Comment:
Missing `unlink($doc_dest);` on cleanup?

Reply:
No, there's nothing to unlink. Instead `$doc_dest` can be removed altogether, since `xmlwriter_open_memory()` doesn't expect any arguments at all.

Reply:
Fixed in https://github.com/php/php-src/commit/d94a1907b735c1c4b0e40a671fa9044c90456c2d. open_memory() was missing a parse_parameters_none() call, so this was silently allowed.